tox.ini

```diff
@@ -34,4 +34,4 @@ deps =
     jupyter
     jupytext
 commands =
-    jupytext --execute docs/tutorial_notebooks/*.ipynb
+    jupytext --execute docs/tutorial_notebooks/*.md
```

Comment:
executing *.ipynb makes some changes to the meta-data in the notebook.
executing *.md works just as well but doesn't leave traces